website/docs/gcp_bigquery.md

```diff
@@ -39,8 +39,9 @@ setting sync tool class. A few BigQuery-specific configurations are required.
 | `hoodie.gcp.bigquery.sync.dataset_location`  | Region info of the dataset; same as the GCS bucket that stores the Hudi table                                   |
 | `hoodie.gcp.bigquery.sync.source_uri`        | A wildcard path pattern pointing to the first level partition; partition key can be specified or auto-inferred. Only required for partitioned tables |
 | `hoodie.gcp.bigquery.sync.source_uri_prefix` | The common prefix of the `source_uri`, usually it's the path to the Hudi table, trailing slash does not matter. |
-| `hoodie.gcp.bigquery.sync.base_path`         | The usual basepath config for Hudi table.                                                                       |
```

Comment:
Is this config removed?

Reply:
that's next- but the sync base path is not usable.

Reply:
I suggest keeping this as it is still in the code base.